docs/src/user_docs/assembly/cryptographic_operations.md

```diff
@@ -11,4 +11,4 @@ Miden assembly provides a set of instructions for performing common cryptographi
 | hmerge  <br> - *(16 cycles)*        | [B, A, ...]     | [C, ...]       | $C \leftarrow hash(A,B)$ <br> where, $hash()$ computes a 2-to-1 Rescue Prime Optimized hash. |
 | mtree_get  <br> - *(9 cycles)*     | [d, i, R, ...]  | [V, R, ...] |  Verifies that a Merkle tree with root $R$ opens to node $V$ at depth $d$ and index $i$. Merkle tree with root $R$ must be present in the advice provider, otherwise execution fails. |
 | mtree_set <br> - *(29 cycles)*      | [d, i, R, V', ...] | [V, R', ...] | Updates a node in the Merkle tree with root $R$ at depth $d$ and index $i$ to value $V'$. $R'$ is the Merkle root of the resulting tree and $V$ is old value of the node. Merkle tree with root $R$ must be present in the advice provider, otherwise execution fails. At the end of the operation the advice provider will contain both Merkle trees. |
-| mtree_cwm <br> - *(29 cycles)*      | [d, i, R, V', ...] | [V, R', ...] | Copies a Merkle tree with root $R$ and updates a node at depth $d$ and index $i$ in the copied tree to value $V'$. $R'$ is the Merkle root of the new tree and $V$ is old value of the node. Merkle tree with root $R$ must be present in the advice provider, otherwise execution fails. At the end of the operation the advice provider will contain both Merkle trees. |
+| mtree_merge <br> - *(12 cycles)*      | [$R_{right}$, $R_{left}$, ...] | [$R_{merged}$, ...] | Merges two Merkle trees with the provided roots $(R_{left}, R_{right})$ into a new Merkle tree with root $R_{merged}$. The input roots are consumed from the operand stack, and the merged root is pushed from the advice stack. |
```

Comment:
I don't think we need to mention the advice stack here.

Also, for stack state, I'm not sure it'll render correctly. We can probably use simple letters to describe the transition:
```
[R, L, ...] -> [M, ...]
```

Reply:
The syntax works, but it's better to keep it concise. Fixed!